axlearn/experiments/testdata/axlearn.experiments.text.gpt.c4_trainer/fuji-7B-single.txt

```diff
@@ -223,7 +221,6 @@ model.decoder.transformer.layer.self_attention.attention.output_linear.param_par
 model.decoder.transformer.layer.self_attention.attention.output_linear.param_partition_spec[0][1]: 'fsdp'
 model.decoder.transformer.layer.self_attention.attention.output_linear.param_partition_spec[0][2]: 'seq'
 model.decoder.transformer.layer.self_attention.attention.output_linear.param_partition_spec[1]: 'model'
-model.decoder.transformer.layer.self_attention.attention.output_linear.param_partition_spec[2]: None
```

Comment:
Applying it to list/tuple/dict seems going too far, e.g., here it will not be easy for readers to tell that `param_partition_spec` actually has three elements rather than two. WDYT?

Reply:
sounds good